lib/common/formats.js

```diff
@@ -25,7 +25,7 @@ function variablesWithPrefix(prefix, properties) {
   return _.map(properties, function(prop) {
       var to_ret_prop = prefix + prop.name + ': ' + (prop.attributes.category==='asset' ? '"'+prop.value+'"' : prop.value) + ';';
 
-      if (prop.attributes.comment)
+      if (prop.comment)
```

Comment:
Should we add this to the javascript/es6 format on line 267 below?

Reply:
yes we should. sorry I missed it*. done.

* re. this I'll open an issue for discussion, about templates vs formats.